integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CarbonScalaUtil.scala

```diff
@@ -158,7 +159,9 @@ object CarbonScalaUtil {
       dataType: DataType,
       timeStampFormat: SimpleDateFormat,
       dateFormat: SimpleDateFormat,
-      serializationNullFormat: String): UTF8String = {
+      serializationNullFormat: String,
+      failAction: Boolean,
+      ignoreAction: Boolean): UTF8String = {
```

Comment:
It is not easy to understand why there is bad record related parameter in this conversion function, can you catch exception in caller and handler it there? There is only one place call this function, and it is better to restrict its scope by moving it there

Reply:
We passed these parameters to throw an exception or ignore it in case of bad records.  It is not good to handle in caller as it is per each column, not for row so caller code will become messy if we try to handle there. 
I will give comments to the parameters for understanding,